weldx/welding/groove/iso_9692_1.py

```diff
@@ -163,6 +187,22 @@ def to_profile(self, width_default: pint.Quantity = Q_(5, "mm")) -> geo.Profile:
 
         return geo.Profile([shape, shape_r], units=_DEFAULT_LEN_UNIT)
 
+    @WELDX_UNIT_REGISTRY.check(None, '[length]', '[length]/[time]', '[length]', '[length]')
+    #TODO: extend the ckeck decorator to check on output! # ret='[length]/[time]')
+    #TODO: thinkness of wire??
+    def compute_welding_speed(self, seam_length: pint.Quantity, wire_feed: pint.Quantity,
+                              width_default: pint.Quantity = Q_(5, 'mm'),
+                              wire_thinkness=Q_(1, 'mm')):
+        # TODO: we need all lengths in same units, otherwise extending area to volume etc does not work...
+        WELDX_UNIT_REGISTRY.auto_reduce_dimensions = True
```

Comment:
I think a final call to `speed.to("mm/s")` or `speed.to_reduced_units()` before return should work as well here